samples/csharp/MicrosoftGraphSDK/Submission.cs

```diff
@@ -96,6 +96,37 @@ public static async Task<EducationSubmissionCollectionResponse> GetSubmissionsWi
             }
         }
 
+        /// <summary>
+        /// List all the submissions associated with an assignment using select
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
+        /// <param name="select">OData query parameter to help customize the response</param>
+        /// <returns>EducationSubmissionCollectionResponse</returns>
+        public static async Task<EducationSubmissionCollectionResponse> GetSubmissionsWithSelectAsync(
+            GraphServiceClient client,
+            string classId,
+            string assignmentId,
+            string select)
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Assignments[assignmentId]
+                    .Submissions
+                    .GetAsync((requestConfiguration) =>
+                    {
+                        requestConfiguration.QueryParameters.Select = new string[] { "status","id" };
```

Comment:
you must use the `select` parameter here, not the properties

Reply:
updated to select and made dependent changes.